book/02-git-basics/sections/aliases.asc

```diff
@@ -1,13 +1,13 @@
 [[_git_aliases]]
-=== Git Aliases
+=== Komut Kısayolu (Alias) Ayarlama
 
 (((aliases)))
-Before we finish this chapter on basic Git, there's just one little tip that can make your Git experience simpler, easier, and more familiar: aliases.
-We won't refer to them or assume you've used them later in the book, but you should probably know how to use them.
+Temel Git ile ilgili bu bölümü bitirmeden önce Git deneyiminizi daha basit, daha kolay ve daha tanıdık hale getirebilecek küçük bir ipucu vereceğiz: komut kısayolları.
+Bu kitapta kısayol (alias) konusuna girmeyeceğiz veya ilerleyen bölümlerde bunları kullandığınızı varsaymayacağız, ancak bunları nasıl kullanacağınızı bilmenizde yine de fayda var.
```

Comment:
"Bu kitapta kısayol (alias) konusuna girmeyeceğiz" orijinalindeki anlamı tam karşılamamış gibi duruyor. Zaten alias chapter'ında "bu kitapta alias konusuna girmeyeceğiz" deyip devamında alias'ı anlatması tuhaf duruyor :)
Sanırım "İlerleyen bölümlerde kısayol (alias) konusuna tekrar değinmeyeceğiz veya bunları kullandığınızı varsaymayacağız, ancak bunları nasıl kullanacağınızı bilmenizde yine de fayda var." gibi bir değişiklik anlamı daha iyi iletir

Reply:
Evet böylesi daha doğru bir çeviri olmuş. Siz gerekli değişikliği yapmışsınız, benim tekrardan bir şey yapmama gerek var mı? Sistemi tam bilmediğim için soruyorum.

Reply:
Ben değişikliği sözlü olarak yapmış oldum 😄 değişikliği yaptıktan sonra 
```bash
git add aliases.asc; git commit --amend; git push --force
``` 
ile aynı commit'e ekleyip push'larsınız commit history daha temiz oluyor, her update için ayrı commit yerine tek commit gözüküyor. 
Dallar ve büyüklükleri hakkında gördüğüm kadarıyla bir standart yok. Ancak büyük olmasını ben kendi adıma tercih etmiyorum çünkü birbirinden habersiz aynı kısımla uğraşıldığında kayıp büyümüş oluyor 😅 mesela burası için sizden sonra bir PR daha açılmış https://github.com/progit/progit2-tr/pull/33 bir de PR ne kadar büyürse review süresi uzadığı için reviewer sayısı o kadar azalıyor. Dalların takibi için de verebileceğim tek tavsiye update/patch gibi isimler yerine chapter2-aliases gibi branch isimleri vermek en azından bulmayı kolaylaştırıyor.

Tabii bunların kalıcı çözümleri için terimler sözlüğü gibi bir issue daha açıp ortak bir sistem konuşulup kararlaşılabilir. Örneğin bir kısımda çeviriye başlayan kişinin yeni bir issue açarak bunu duyurması, branch isimleri, commit mesajları, PR büyüklükleri vb